lineman/app/components/memberships_page/cancel_invitation_form/cancel_invitation_form.coffee

```diff
@@ -0,0 +1,9 @@
+angular.module('loomioApp').factory 'CancelInvitationForm', ->
+  templateUrl: 'generated/components/cancel_invitation_form/cancel_invitation_form.html'
+  controller: ($scope, invitation, FlashService, Records) ->
+    $scope.invitation = invitation
+
+    $scope.submit = ->
```

Comment:
I believe the FormService is merged in, so let's use that instead. There's a bunch of examples in the other forms now.